python/IAM_POLICY_REQUIRED/IAM_POLICY_REQUIRED.py

```diff
@@ -0,0 +1,526 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+"""
+#####################################
+##           Gherkin               ##
+#####################################
+ Rule Name:
+   IAM_POLICY_REQUIRED
+ Description:
+   To check IAM users and roles have a given policy attached directly or through a group.
+ Trigger:
+   Configuration Change on AWS::IAM::User/AWS::IAM::Role
+ Reports on:
+   AWS::IAM::User,AWS::IAM::Role
+ Rule Parameters:
+   | ---------------------- | --------- | -------------------------------------------------------- |
+   | Parameter Name         | Type      | Description                                              |
+   | ---------------------- | --------- | -------------------------------------------------------- |
+   | policyArns             | Required  | ARNs of the policies which should be attached to all     |

```

Comment:
Please add an example in the description. It seems that you allow list and/or string. I'd settle on one or this other personally, no need to support both.

Reply:
no problem, will go with list